x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_settings_collector.ts

```diff
@@ -44,21 +47,35 @@ interface EmailSettingData {
   xpack: { default_admin_email: string | null };
 }
 
-export interface KibanaSettingsCollector extends Collector<EmailSettingData | undefined> {
-  getEmailValueStructure(email: string | null): EmailSettingData;
+export type KibanaSettingsCollector = Collector<EmailSettingData | undefined>;
+
+export function getEmailValueStructure(email: string | null) {
+  return {
+    xpack: {
+      default_admin_email: email,
+    },
+  };
 }
 
-export function getSettingsCollector(usageCollection: any, config: MonitoringConfig) {
+export function getSettingsCollector(
+  usageCollection: UsageCollectionSetup,
+  config: MonitoringConfig
+) {
   return usageCollection.makeStatsCollector({
     type: KIBANA_SETTINGS_TYPE,
     isReady: () => true,
-    async fetch(this: KibanaSettingsCollector) {
+    schema: {
+      xpack: {
+        default_admin_email: { type: 'text' },
+      },
+    },
+    async fetch(this: Collector<EmailSettingData | undefined>) {
```

Comment:
TS's fancy way of specifying the expected type for `this`. It doesn't affect the arguments.